lib/ModuleBuildError.js

```diff
@@ -12,8 +12,13 @@ class ModuleBuildError extends WebpackError {
 		super();
 
 		this.name = "ModuleBuildError";
-		this.message = "Module build failed: ";
+		this.message = "Module build failed";
 		if(err !== null && typeof err === "object") {
+			if(err.from) {
+				this.message += ("(from " + err.from + ")");
```

Comment:
The brackets around the string are not necessary.

Use template literals.

Add a space between `Module build failed` and `(from` => `Module build failed (from babel-loader):`